src/util/toggle-class.js

```diff
@@ -0,0 +1,38 @@
+//
+// helper to add/remove classes for IE9
+//
+// Usage:
+//
+// toggleClass(div, 'demo'); // Toggles the class `demo`
+// toggleClass(div, 'demo', true); // Adds the class `demo`
+// toggleClass(div, 'demo', false); // removes the class `demo`
+//
+export default function toggleClass(element, className, toggleClass) {
```

Comment:
the parameter should be called the same way the function is. I'd suggest calling this option `force`